### ISSUE TITLE
[package] qt/6.6.0: Version conflict: freetype/2.13.2 and freetype/2.13.0

### ISSUE THREAD

**Issue by ashley-b** [conan-io/conan-center-index#21286]
### Description

Trying to create qt package from master fails
`cd recipes/qt/6.x.x && conan create . --version  6.6.0`

Returns
```
ERROR: Version conflict: Conflict between freetype/2.13.2 and freetype/2.13.0 in the graph.
Conflict originates from fontconfig/2.14.2
```

### Package and Environment Details

* Package Name/Version: **qt/6.6.0**
* Operating System+version: **Linux Ubuntu 18.04**
* Compiler+version: **GCC 8**
* Docker image: **conanio/gcc8**
* Conan version: **Conan version 2.0.14**
* Python version: **Python 3.7.13**

### Conan profile

Host profile:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=13
os=Linux

Build profile:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=13
os=Linux

### Steps to reproduce

From master
`cd recipes/qt/6.x.x && conan create . --version  6.6.0`

### Logs

<details><summary>Click to expand log</summary>

```
#9 [6/6] RUN cd recipes/qt/6.x.x && conan create . --version  6.6.0
#9 2.502 
#9 2.502 ======== Exporting recipe to the cache ========
#9 2.590 qt/6.6.0: Exporting package recipe: /home/conan/conan-center-index/recipes/qt/6.x.x/conanfile.py
#9 2.591 qt/6.6.0: exports: File 'conandata.yml' found. Exporting it...
#9 2.593 qt/6.6.0: Calling export()
#9 2.595 qt/6.6.0: Calling export_sources()
#9 2.598 qt/6.6.0: Copied 1 '.conf' file: qtmodules6.6.0.conf
#9 2.598 qt/6.6.0: Copied 1 '.py' file: conanfile.py
#9 2.598 qt/6.6.0: Copied 1 '.yml' file: conandata.yml
#9 2.598 qt/6.6.0: Copied 1 '.diff' file: c72097e_6.6.0.diff
#9 2.601 qt/6.6.0: Exported to cache folder: /home/conan/.conan2/p/qt39f7933ec1ea2/e
#9 2.601 qt/6.6.0: Exported: qt/6.6.0#1e674e88be66ff45a029c62957b7b6f9 (2023-11-21 05:04:21 UTC)
#9 2.601 
#9 2.601 ======== Input profiles ========
#9 2.601 Profile host:
#9 2.601 [settings]
#9 2.601 arch=x86_64
#9 2.601 build_type=Release
#9 2.601 compiler=gcc
#9 2.601 compiler.cppstd=gnu14
#9 2.601 compiler.libcxx=libstdc++11
#9 2.601 compiler.version=8
#9 2.601 os=Linux
#9 2.601 
#9 2.601 Profile build:
#9 2.601 [settings]
#9 2.601 arch=x86_64
#9 2.601 build_type=Release
#9 2.601 compiler=gcc
#9 2.601 compiler.cppstd=gnu14
#9 2.601 compiler.libcxx=libstdc++11
#9 2.601 compiler.version=8
#9 2.601 os=Linux
#9 2.601 
#9 2.606 
#9 2.606 ======== Computing dependency graph ========
#9 3.224 zlib/1.3: Not found in local cache, looking in remotes...
#9 3.224 zlib/1.3: Checking remote: conancenter
#9 3.555 zlib/1.3: Downloaded recipe revision 06023034579559bb64357db3a53f88a4
#9 3.813 openssl/3.1.4: Not found in local cache, looking in remotes...
#9 3.813 openssl/3.1.4: Checking remote: conancenter
#9 3.877 openssl/3.1.4: Downloaded recipe revision 88e860e93e58ec156d1d3b08c643427a
#9 3.895 pcre2/10.42: Not found in local cache, looking in remotes...
#9 3.895 pcre2/10.42: Checking remote: conancenter
#9 4.170 pcre2/10.42: Downloaded recipe revision 74a354d74d291503aadffe6453d976f5
#9 4.182 bzip2/1.0.8: Not found in local cache, looking in remotes...
#9 4.182 bzip2/1.0.8: Checking remote: conancenter
#9 4.247 bzip2/1.0.8: Downloaded recipe revision 411fc05e80d47a89045edc1ee6f23c1d
#9 4.256 double-conversion/3.3.0: Not found in local cache, looking in remotes...
#9 4.256 double-conversion/3.3.0: Checking remote: conancenter
#9 4.537 double-conversion/3.3.0: Downloaded recipe revision 33321c201741cc32b51169c6d2d05e60
#9 4.548 freetype/2.13.0: Not found in local cache, looking in remotes...
#9 4.548 freetype/2.13.0: Checking remote: conancenter
#9 4.831 freetype/2.13.0: Downloaded recipe revision 28bb2425d43c8bf7dbcfb024d847f5fb
#9 4.845 libpng/1.6.40: Not found in local cache, looking in remotes...
#9 4.845 libpng/1.6.40: Checking remote: conancenter
#9 4.906 libpng/1.6.40: Downloaded recipe revision 0337b7b87ac125307ed70fd064c31eea
#9 4.918 brotli/1.1.0: Not found in local cache, looking in remotes...
#9 4.918 brotli/1.1.0: Checking remote: conancenter
#9 5.211 brotli/1.1.0: Downloaded recipe revision d56d7bb9ca722942aba17369cb5c0519
#9 5.222 fontconfig/2.14.2: Not found in local cache, looking in remotes...
#9 5.222 fontconfig/2.14.2: Checking remote: conancenter
#9 5.290 fontconfig/2.14.2: Downloaded recipe revision e43d6947356e29dee0ab8b05f75bd900
#9 5.306 Graph root
#9 5.306     cli
#9 5.306 Requirements
#9 5.306     brotli/1.1.0#d56d7bb9ca722942aba17369cb5c0519 - Downloaded (conancenter)
#9 5.306     bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d - Downloaded (conancenter)
#9 5.306     double-conversion/3.3.0#33321c201741cc32b51169c6d2d05e60 - Downloaded (conancenter)
#9 5.306     fontconfig/2.14.2#e43d6947356e29dee0ab8b05f75bd900 - Downloaded (conancenter)
#9 5.306     freetype/2.13.0#28bb2425d43c8bf7dbcfb024d847f5fb - Downloaded (conancenter)
#9 5.306     libpng/1.6.40#0337b7b87ac125307ed70fd064c31eea - Downloaded (conancenter)
#9 5.306     openssl/3.1.4#88e860e93e58ec156d1d3b08c643427a - Downloaded (conancenter)
#9 5.306     pcre2/10.42#74a354d74d291503aadffe6453d976f5 - Downloaded (conancenter)
#9 5.306     qt/6.6.0#1e674e88be66ff45a029c62957b7b6f9 - Cache
#9 5.306     zlib/1.3#06023034579559bb64357db3a53f88a4 - Downloaded (conancenter)
#9 5.306 Resolved version ranges
#9 5.306     openssl/[>=1.1 <4]: openssl/3.1.4
#9 5.306     zlib/[>=1.2.11 <2]: zlib/1.3
#9 5.309 ERROR: Version conflict: Conflict between freetype/2.13.2 and freetype/2.13.0 in the graph.
#9 5.309 Conflict originates from fontconfig/2.14.2
#9 5.309 
#9 5.309 Run conan graph info with your recipe and add --format=html to inspect the graph errors in an easier to visualize way.
#9 ERROR: process "/bin/sh -c cd recipes/qt/6.x.x && conan create . --version  6.6.0" did not complete successfully: exit code: 1

```

</details>

**Comment by ericLemanissier** [comment]
Fixed in https://github.com/conan-io/conan-center-index/pull/21263

**Comment by SpaceIm** [comment]
Also in https://github.com/conan-io/conan-center-index/pull/21223

**Comment by ericLemanissier** [comment]
This PR can be closed now that https://github.com/conan-io/conan-center-index/pull/21263 is merged

**Comment by ViktarHasiul231862** [comment]
I guess version conflict in freetype lib still remains for Qt5.

**Comment by ashley-b** [comment]
@ViktarHasiul231862 
> I guess version conflict in freetype lib still remains for Qt5.

Fixed in #21362
